src/quantum/plugin/FlatFileGenerator.ts

```diff
@@ -68,25 +68,23 @@ export class FlatFileGenerator {
                 }
             }
         }
-        if (this.entryId !== undefined) {
-            const req = `${this.core.opts.quantumVariableName}.r(${JSON.stringify(this.entryId)})`;
 
-            if (this.globalsName) {
-                if (this.core.opts.isTargetNpm() || this.core.opts.isTargetServer()) {
-                    this.contents.push(`module.exports = ${req}`);
-                }
+        this.globals.forEach((fileID, globalName) => {
+            const req = `${this.core.opts.quantumVariableName}.r(${JSON.stringify(fileID)})`;
+            if (this.core.opts.isTargetNpm() || this.core.opts.isTargetServer()) {
+                this.contents.push(`module.exports = ${req}`);
```

Comment:
It's not gonna work for module.exports here
We need to have only one module exports for server

Reply:
Also please check the tests

Reply:
They all need to pass, before we write new ones

Reply:
Fixed, also brought back in use of "*"